db/mysql/client.go

```diff
@@ -10,14 +10,15 @@ import (
 
 	"github.com/ahab94/ApnaSchool/db"
 	"github.com/ahab94/ApnaSchool/models"
+	"github.com/ahab94/ApnaSchool/configuration"
 )
 
 const (
-	databaseName     = "apna_school"
+	databaseName     = configuration.DbName
 	studentTableName = "student"
 	teacherTableName = "teacher"
-	databasePassword = "Talha1996@gmail.com"
-	databaseUser     = "talha"
+	databasePassword = configuration.DbPass
```

Comment:
this should be `viper.GetString(configuration.DbPass)` same for other occassions.

remember `configration.DbPass` is just a key. To access it's value which we stored in the viper.. we do
`viper.GetString(configuration.DbPass)`